include/engine/datafacade/shared_datafacade.hpp

```diff
@@ -79,7 +79,7 @@ class SharedDataFacade final : public BaseDataFacade
 
     util::ShM<util::Coordinate, true>::vector m_coordinate_list;
     util::PackedVector<OSMNodeID, true> m_osmnodeid_list;
-    util::ShM<NodeID, true>::vector m_via_node_list;
+    util::ShM<extractor::GeometryID, true>::vector m_via_node_list;
```

Comment:
again, referring to nodes by a geometry ID. They are still node ids, right? Each node also has a coordinate, though.